### ISSUE TITLE
Make "View Larger" PDF button a link and change text

### ISSUE THREAD

**Issue by lschiff** [eScholarship/jschol#965]
The "View Larger" button on item pages, which triggers a full browser view of the PDF, doesn't meet accessibility requirements.  See item 23 in the [accessibility issues report](https://docs.google.com/spreadsheets/d/1kIMiW09tBtk5d3kJjzUMSCo1tEWPn7bUc0BYAJes9Hw/edit?gid=0#gid=0) 

Two changes are required:

1. Make the button a link.
2. The link text should read "Open PDF in current tab"

**Comment by nicktahani** [comment]
@justingonder @lschiff  I think opening this in a new tab would be better UX and resolve the issue of an unexpected change of context. Any thoughts on the button labeling?
